src/trace/dma-trace.c

```diff
@@ -328,6 +332,9 @@ int dma_trace_enable(struct dma_trace_data *d)
 		goto out;
 #endif
 
+	/* flush fw description message */
+	trace_flush();
+
```

Comment:
@ktrzcinx Why, with this change, tr_info and trace_flush() are no longer close one to each other?

Reply:
To flush after `dma_trace_start` - possible part of trace initialization while writing trace log just after buffer initialization (after moving it one line below then it's not the first trace log)

Reply:
Ok.